analyzers/dataframe/FCCAnalyses/JetClusteringUtils.h

```diff
@@ -4,6 +4,8 @@
 
 #include "Math/Vector4D.h"
 #include "ROOT/RVec.hxx"
+#include "TLorentzVector.h"
```

Comment:
This header seems to be used only in the implementation file. Can it be moved there?

Reply:
yes, done